x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_source_logic.test.ts

```diff
@@ -353,6 +358,30 @@ describe('AddSourceLogic', () => {
         expect(navigateToUrl).toHaveBeenCalledWith(`${ADD_GITHUB_PATH}/configure${queryString}`);
       });
 
+      describe('Github error edge case', () => {
+        it('handles "organization" redirect and displays error', () => {
+          const githubQueryString =
+            '?error=redirect_uri_mismatch&error_description=The+redirect_uri+MUST+match+the+registered+callback+URL+for+this+application.&error_uri=https%3A%2F%2Fdocs.github.com%2Fapps%2Fmanaging-oauth-apps%2Ftroubleshooting-authorization-request-errors%2F%23redirect-uri-mismatch&state=%7B%22action%22%3A%22create%22%2C%22context%22%3A%22organization%22%2C%22service_type%22%3A%22github%22%2C%22csrf_token%22%3A%22TTAyYktJSkU5Um9PQ0p5clNEbGk0MnM0cXhqMXZXaDBxdG8zeXQ3YkRnYkIyempWaUMwMGpuZmpLR25pcHRKT0RxUXFvbTJGM20vZE5RcUpEcVhWNEE9PQ%3D%3D%22%2C%22index_permissions%22%3Afalse%7D';
```

Comment:
**Question:** Is this token safe to store here?

Reply:
https://github.com/elastic/kibana/pull/105302/commits/718d4bbb99a3df4f73d24c66ff5495ecc6220fcb